utils/jsonSigning.js

```diff
@@ -41,8 +55,48 @@ module.exports.sign = function (json, prvKey, pubKey) {
   return signedJSON
 }
 
+// Sign JSON object using RSA
+function signRSA (json, prvKey) {
+  if (!prvKey) throw new ServerError('Problem reading private key for JSON signing')
+  let prvKeyObj = null
+  // If given private key is PEM formatted string, convert it to private key object
+  // If it's already private key object, just use it
+  if (typeof prvKey === 'string') {
```

Comment:
Looks like this logic is also used in `verify` to get the public key. Might be worth extracting to a helper method and then: `const prvKeyObj = getKey(prvKey)`, and similarly with `pubKeyObj`.